sample-customizations/custom-carbon-log-appender/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
@vihanga-liyanage I do not think we need to include this license again. We already  have one in root level of sample-is